lib/elixir/test/elixir/access_test.exs

```diff
@@ -86,4 +86,25 @@ defmodule AccessTest do
       Access.pop(struct(Sample, []), :name)
     end
   end
+
+  describe "filter/1" do
+    @test_list [1, 2, 3, 4, 5, 6]
+
+    test "filters in get_in" do
+      assert(get_in(@test_list, [Access.filter(&(&1 > 3))]) == [4, 5, 6])
```

Comment:
Can you please remove the parens around `assert` on this test and the ones below?